src/components/Explore/Modals/ExploreFiltersModal.tsx

```diff
@@ -1,22 +1,21 @@
-// @flow
-
 import Modal from "components/SharedComponents/Modal";
-import type { Node } from "react";
 import React from "react";
 
 import FilterModal from "./FilterModal";
 
-type Props = {
-  showModal: boolean,
-  closeModal: Function,
-  filterByIconicTaxonUnknown: Function,
-  renderLocationPermissionsGate: Function,
-  requestLocationPermissions: Function,
-  updateTaxon: Function,
-  updateLocation: Function,
-  updateUser: Function,
-  updateProject: Function
-};
+interface Props extends Pick<
+  React.ComponentProps<typeof FilterModal>,
```

Comment:
I'm not sure we necessarily want to do this type of type manipulation here to migrate this file. This definitely helps identify that we have an opportunity for type reuse among these similar / related components in this file / directory. That said, I'd say let's go for duplication _at the moment_ because it's in line with the other files.

We're still talking through how we want to handle shared type definition / shared type extraction but I think in general, that will be more holistically addressed after we have just fewer `js` files.